eZ/Publish/Core/Persistence/Legacy/Content/FieldValue/Converter/ISBNConverter.php

```diff
@@ -82,7 +82,7 @@ public function toFieldDefinition(StorageFieldDefinition $storageDef, FieldDefin
     {
         $fieldDef->fieldTypeConstraints->fieldSettings = new FieldSettings(
             array(
-                'isISBN13' => !empty($storageDef->dataInt1) ? (bool)$storageDef->dataInt1 : true,
+                'isISBN13' => !empty($storageDef->dataInt1),
```

Comment:
So while this is correct, it still is a bit gotcha reading.

Would it make sense to make it more explicit so it's instantly what the code does here?

Like for instance: `$storageDef->dataInt1 !== null ? (bool)$storageDef->dataInt1 : true;`